public/index.html

```diff
@@ -24,7 +22,40 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+
+    <link rel="canonical" href="https://hyper.leet.ar/" />
+
+    <link rel="icon" href="%PUBLIC_URL%/images/favicon.ico">
+    <link rel="shortcut icon" href="%PUBLIC_URL%/images/favicon.ico">
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/images/hypercore-protocol.png" />
+    <link rel="apple-touch-icon-precomposed" href="%PUBLIC_URL%/images/hypercore-protocol.png">
+    
+    <title>Hyperfind</title>
+
+    <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <meta name="google" value="notranslate" />
+    <meta name="robots" content="noindex, nofollow" />
+
+    <meta name="language" content="en" />
+    <meta name="title" content="Hyperfind" />
+    <meta name="description" content="" />
```

Comment:
`Discover Hypercores, and seed your own cores`

Después vemos de mejorar la descripción